lua/pac3/core/client/parts/animation.lua

```diff
@@ -218,6 +247,41 @@ function PART:OnShow()
 	end
 end
 
+-- Play animation and move to top of animation stack
+function PART:OnShow()
+	local ent = self:GetOwner()
+	if ent:IsValid() then
+		local stack = ent.pac_animation_stack
+		if stack then
+			local count = #stack
+			if count == 0 then
+				-- Empty stack
+				table.insert(stack, self)
```

Comment:
If you use #count operator to check if table is not empty, then you also can use it to insert new value into it (`stack[count + 1] = self`)